opal/mca/btl/usnic/btl_usnic_module.c

```diff
@@ -1436,6 +1438,21 @@ static int create_ep(opal_btl_usnic_module_t* module,
         channel->info->caps &= ~(1ULL << 63);
     }
 
+    /* all of the OMPI code assumes IPv4, but some versions of libfabric will
+     * return FI_SOCKADDR instead of FI_SOCKADDR_IN, so we need to do a little
+     * bit of sanity checking */
+    assert(FI_SOCKADDR_IN == channel->info->addr_format ||
+           FI_SOCKADDR == channel->info->addr_format);
+    if (FI_SOCKADDR == channel->info->addr_format) {
```

Comment:
Disregard my previous comment... I see that the purpose of this "if" is mainly just another assert.

However, it'll generate a CID / compiler warning in non-debug builds, because sa will be assigned and not used, right?